docs/build/tooling/hardhat/hardhat-zksync-solc.md

```diff
@@ -109,6 +109,32 @@ For Solidity versions older than 0.8, only this compilation mode is available an
 
 :::
 
+For usage of zkVm compiler, `eraVersion` should be added inside `solidity` property in the `hardhat.config.ts` file:
```

Comment:
zkVm -> zkVM?

Should we now rename zkEVM and EraVM to zkVM everywhere?

Reply:
Hello @hedgar2017,

I wanted to inform you that we have made a transition from zkVM to EraVM, as documented in the [official repository] (https://github.com/matter-labs/compiler-solidity). Please review the updated information to ensure that all details are accurate. Thanks!

Reply:
@kiriyaga-txfusion that repo is already archived and obsolete in favor of [the open-source one](https://github.com/matter-labs/era-compiler-solidity).
However, we still haven't synced across the whole docs regarding this naming, so I leave this decision to the doc owners.